tests/renderer/investtab.test.js

```diff
@@ -61,6 +63,7 @@ describe('Sidebar Alert renders with data from a recent run', () => {
     fetchValidation.mockResolvedValue([]);
     const mockSpec = spec; // jest.mock not allowed to ref out-of-scope var
     jest.mock(UI_CONFIG_PATH, () => mockUISpec(mockSpec));
+    setupDialogs();

```

Comment:
Now that the electron api call to open dialogs was moved to the main process, the code under test here is making an IPC call where previously it was not. So that's why we need to setup and remove the listeners in this block. A couple of the tests are clicking on the "Open Workspace" button.